docs/usage/executor/x.md

```diff
@@ -0,0 +1,84 @@
+---
+id: x
+title: "X plugin"
+sidebar_position: 3
+---
+
+The `x` executor plugin allows you to install and run CLIs directly from a chat window (e.g. Slack, Discord) without any hassle.
+
+:::tip
+Interactivity in the form of buttons and dropdowns is also supported. It's crucial as you can easily run all your favourite tools against Kubernetes directly from your phone too.
+:::
+
+![x-demo](./assets/demo-x.gif)
+
+# Usage
+
+## Install CLI
+
+To install a given CLI binary directly from a chat window, run:
+
+```
+@Botkube x install https://get.helm.sh/helm-v3.10.3-linux-amd64.tar.gz --file helm
+```
+
+## Execute CLI
+
+To execute a CLI command, run:
+
+```
+@Botkube x run helm list -A
+```
+
+:::tip
+For convenience, you can configure a Botkube alias, to simplify running `helm` commands:
+
+```yaml
+aliases:
+  helm:
+    command: x run helm
+    displayName: "Helm CLI alias"
+```
+
+Later you can type just, `@Botkube helm list -A`.
+:::
```

Comment:
While I totally understand the idea here, I think it might be confusing to list `helm` as well. WDYT about section like `Examples` and list there sample commands like helm, flux, argo, etc.? 2-3 of them for which you already prepared templates. Even alias example could be a bit different (e.g. argo CLI?) to avoid confusion with our current Helm plugin. 

Also, please describe the installation syntax a bit better (possible flags, url - what we use underneath; we can point to external docs for the dependency we use)

Something like:

``````md
## Install CLI

To install a given CLI binary directly from a chat window, run:

```
@Botkube x install {source}
```

TODO: Describe how the source should look like (point to the docs to our dependency etc.)

### Examples

- `@Botkube x install https://get.helm.sh/helm-v3.10.3-linux-amd64.tar.gz --file helm`
- ...

## Execute CLI

To execute a CLI command, run:

```
@Botkube x run {command}
```

### Examples

- `@Botkube x run helm list -A`
- ...
``````